sprotty-vscode-extension/package.json

```diff
@@ -42,10 +42,11 @@
   "main": "lib/index",
   "types": "lib/index",
   "dependencies": {
+    "lerna": "^3.20.2",
```

Comment:
`lerna` is a build tool and should not be in `dependencies`.

Reply:
✅